app/management/commands/update_ungm.py

```diff
@@ -119,14 +122,17 @@ def update_ungm_tenders(parsed_tenders):
                                 setattr(tender_doc, k, v)
 
                         tender_doc.save()
+                        self.download_document(tender_doc)
 
                     except TenderDocument.DoesNotExist:
-                        TenderDocument.objects.create(tender=tender_item, **doc)
+                        new_doc = TenderDocument.objects.create(tender=tender_item, **doc)
+                        self.download_document(new_doc)
```

Comment:
Call download_document from class : Command.download_document as it is a static method